cypress-tests/cypress/support/utils/inspector.js

```diff
@@ -32,3 +32,8 @@ export const verifyValue = (node, type, children, index = 0) => {
     .eq(index)
     .verifyVisibleElement("have.text", type);
 };
+export const deleteComponentFromInspector = (node) => {
+  cy.get('[data-cy="inspector-node-components"] > .node-key').click();
+  cy.get(`[data-cy="inspector-node-${node}"] > .node-key`).click();
+  cy.get(cy.get('[style="height: 13px; width: 13px;"] > img').click());
```

Comment:
I think we can only use `cy.get('[style="height: 13px; width: 13px;"] > img').click()` here.